app/med-list-create/med-list-create.ts

```diff
@@ -0,0 +1,10 @@
+import {DoseMedication} from "../dose-medication/dose-medication";
+
+export class MedListCreate {
```

Comment:
`MedListCreate` -- What is this object?

There is nothing on the server that will match this. We just have `Med`, which maps to client-side `DoseMedication`.
https://github.com/dose-amigos/dose-amigos/wiki/Data-Objects-we'll-need

What do you envision we will do with this?